src/pages/workspaces/workspace/Data.js

```diff
@@ -640,7 +639,7 @@ class EntitiesContent extends Component {
 
     const header = _.join('\t', [`entity:${entityKey}_id`, ...attributeNames])
 
-    const entityTsv = _.join('\n', [header, ..._.map(entityToRow, sortedEntities)]) + '\n'
+    const entityTsv = `${_.join('\n', [header, ..._.map(entityToRow, sortedEntities)])}\n`
```

Comment:
What about this for these?
```suggestion
    const entityTsv = _.join('\n', [header, ..._.map(entityToRow, sortedEntities), ''])
```

Reply:
FWIW that seems less clear to me.

Reply:
kept this as-is, if there is a strong objection to changing it let me know